advisor/src/test/assets/ort-analyzer-result.yml

```diff
@@ -571,4 +571,8 @@ analyzer:
 scanner: null
 advisor: null
 evaluator: null
-resolved_configuration: {}
+resolved_configuration:
```

Comment:
Commit message: There is a difference between footnotes and bottom links. For links use `[1]`, for footnotes use `[^1]`. Also, there is a colon missing after `[^1]`.

Reply:
They should not be. For bottom links GitHub creates a link directly to the URL, for footnotes GitHub creates an anchor link so it just scrolls down a little bit to where the footnote is (actually it always scrolled too far for me).